src/base/bittorrent/session.cpp

```diff
@@ -3752,6 +3755,11 @@ bool Session::validateHTTPSTrackerCertificate() const
     return m_validateHTTPSTrackerCertificate;
 }
 
+bool Session::mitigateSsrf() const
```

Comment:
You should group it together with the setter function and place it below the `ValidateHTTPSTrackerCertificate` group.

Reply:
Also you should write SSRF whenever possible. E.g. `mitigateSSRF()`.